docs/software_setup/docker.md

```diff
@@ -46,13 +89,47 @@ sudo systemctl enable docker   # enable docker daemon at boot
 ./enable.sh $HOME   # we'll use home directory as working directory, set this as you please.
 ```
 
+???+ info
+    If you are testing a repo/branch that has containers images hosted on NGC, you need to log into the NGC rfegistry first.
+    
+    Log into NGC, and go to the [Setup > API Key](https://ngc.nvidia.com/setup/api-key) section.
+    If you have not generated your API Key, click on the button "**Generate API Key**" at the right upper corner of the page, and save the API key for the future use.
+    
+    Then, enter the following command on your Jetson and follow the prompts.
+
+    ```bash
+    $ sudo docker login nvcr.io
+    
+    Username: $oauthtoken
+    Password: <You Key>
+    ```
+
+    For the username, enter `$oauthtoken` exactly as shown. It is a special authentication token for all users.
+
+
 Now you can go to ``https://<jetbot_ip>:8888`` from a web browser and start programming JetBot!
+
 You can do this from any machine on your local network.  The password to log in is ``jetbot``.
 
 ![](https://user-images.githubusercontent.com/25759564/92091965-51ae4f00-ed86-11ea-93d5-09d291ccfa95.png)
 
 
-> Note: The directory you specify to ``./enable.sh`` will be mounted as a volume in the jupyter container 
-at the location ``/workspace``.  This means the work you in the ``/workspace`` folder inside container
-is saved.  This is set to the root directory of Jupyter Lab.  Please note, if you work outside of that directory it will be lost when the container shuts down.
+???+ caution
```

Comment:
I think the only thing the user needs to know at this basic setup stage, is that the PiOLED will display the IP address at boot.  So after the web browser image, this should read:

---

The ``enable.sh`` script causes the docker containers to restart at boot.  This means the next time you power on your JetBot, the containers will automatically start, and you should see the IP address displayed on the PiOLED display screen.  All you need to do is type this into your web browser and start programming!

???+ tip 
   For more information on configuring and using docker with JetBot, check out the [Docker Tips](reference/docker_tips.md) page.

---

Also, this contains a lot of information not-related to basic getting started flow.  Too many admonitions here, some of which are redundant.  Let's move these to "refernece/docker_tips.md" page as non-admonitions:


# Docker Tips
...
## Using custom Jupyter Lab workspace directory

The argument you provide to the ``enable.sh`` script is mounted as a volume inside the docker container.  Inside the docker container, this volume is located at ``/workspace``.  Any changes that you make inside this directory will be saved outside of the docker container.  To use a different directory as the workspace for the Jupyter contianer, just provide the path as an argument when calling ``enable.sh``.  For example, to set the workspace to your home directory, you would do

```
cd jetbot/docker
./enable.sh $HOME
```


## Disabling containers

Once you call the ``enable.sh`` script, the docker containers are configured to restart automatically.  This means you can shut down your Jetson, and when you reboot the containers will run and you don't need to repeat this process.  To prevent the containers from starting automatically, just call the ``disable.sh`` script.

```bash
cd jetbot/docker
./disable.sh
```

Reply:
Moved to reference/docker_tips.md